torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1862,21 +1862,44 @@ def sample_inputs_take_along_dim(op_info, device, dtype, requires_grad, **kwargs
             )
 
 def sample_inputs_amax_amin(op_info, device, dtype, requires_grad, **kwargs):
-    # Ordered as (shape, positional args, kwargs)
-    test_cases: Tuple[tuple, tuple, dict] = (  # type: ignore[assignment]
-        ((S, S, S), (), {}),
-        ((S, S, S), (1,), {}),
-        ((S, S, S), ((1, 2,),), {}),
-        ((S, S, S), (1,), {'keepdim': True}),
-        ((), (0,), {}),
-        ((), (), {}),
-        ((), (0,), {'keepdim': True}),
+    # Ordered as (input shape, kwargs)
+    test_cases: Tuple[Tuple[int, ...], Dict[str, Any]] = (  # type: ignore[assignment]
```

Comment:
I think the lint build is unhappy:

```
torch/testing/_internal/common_methods_invocations.py:1866:50: error: Name 'Any' is not defined  [name-defined]
```

I wouldn't bother typing your code here. If the lint build is finicky about typing for some reason then we should fix it to understand this file is untyped. But that's really for a separate PR so you can just tell lint to ignore lines it flags.

Reply:
I think it's just missing an import. VSCode has been imperfect at auto importing things for me haha.